### ISSUE TITLE
Bug: axe-core/cli validator does not report missing "aria-controls" attribute from element with role "scrollbar" (ARIA 1.2)

### ISSUE THREAD

**Issue by spectranaut** [dequelabs/axe-core-npm#318]
**Product:** cli

**Motivation:** 

ARIA 1.2 introduces the following imperative in the [role scrollbar definition](https://www.w3.org/TR/wai-aria-1.2/#scrollbar): `Authors MUST set the aria-controls attribute on the scrollbar element to reference the scrollable area it controls.`

**Replication of error:** 

If you run the AXE cli (axe-core 4.3.2 in chrome-headless with `--tags cat.aria`) on an HTML with the following HTML you will see no errors.

```
<div id="scrollbar" role="scrollbar" aria-valuenow="10"></div>

<div id="id1"></div>
```

**Expectation:**

You should see errors for the scrollbar element because it is missing the "aria-controls" attribute.

**Comment by Zidious** [comment]
Thank you for finding this issue. 

The issue is within [axe-core](https://github.com/dequelabs/axe-core). Currently due to lack of screen-reader support across multiple browsers it is not denoted as a required attribute but a support attribute. 

I have opened this within axe-core see: https://github.com/dequelabs/axe-core/issues/3146

Thank you.
